model/glove/cofreq.go

```diff
@@ -0,0 +1,82 @@
+// Copyright © 2017 Makoto Ito
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package glove
+
+import (
+	"math"
+	"math/rand"
+
+	"github.com/chewxy/lingo/corpus"
+)
+
+// CofreqMap stores the co-frequency between word-word.
+type CofreqMap map[Pair]float64
+
+// Pair stores the co-frequency pair words.
```

Comment:
To make this even faster check this out: https://blog.chewxy.com/2017/07/12/21-bits-english/. I'll help you convert when I find the time

Reply:
I'll watch it, thanks!